tests/integration/config.py

```diff
@@ -0,0 +1,8 @@
+# Paths and directories used in the integration test. Configurations in
+# `all_sets.cfg` and `all_sets_modified.cfg` should match these paths.
+TEST_ROOT_PATH = "tests/integration/"
+TEST_DATA_DIR = "integration_test_data"
+TEST_IMAGES_DIR = "integration_test_images"
+
+TEST_DATA_PATH = f"{TEST_ROOT_PATH}/{TEST_DATA_DIR}/"
+TEST_IMAGES_PATH = f"{TEST_ROOT_PATH}/{TEST_IMAGES_DIR}/"
```

Comment:
This helps with reducing code redundancy throughout the integration test files.

There are probably effective ways to do this (e.g., using env vars), but it is more time consuming. If desired, it can be addressed in a future issue.